solutions/scripts/avn-power-services.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# jq needs to be installed https://stedolan.github.io/jq/
+# The PROJECT services needs to have at least these 2 tags
+#        { "env": ENVIRONMENT,
+#    "start-seq": SEQUENCE_NUMBER }
+#      ENVIRONMENT could be anything like "test1", "PROD", "dev", etc
+#  SEQUENCE_NUMBER should be an integer like 10, 50, 130, etc
+# avn cli needs to be authenticated
+#  https://console.aiven.io/profile/auth "Authentication tokens" -> [Generate token]
+#  avn user login --token [email]
+#
+# This file follows these bash/sh shell scripts rules https://github.com/koalaman/shellcheck
+
+USAGE="usage: --project PROJECT --tag-env ENVIRONMENT [--on | --off]"
+if [ $# -ne 5 ]
+then
+  echo "$USAGE"
+  exit 1
+fi
+
+while [ $# -gt 0 ]; do
+  case $1 in
+    --project)
+      shift && PROJECT="$1"
+      ;;
+    --tag-env)
+      shift && TAG_ENV="$1"
+      ;;
+    --on)
+      shift && POWER="ON"
+      ;;
+    --off)
+      shift && POWER="OFF"
+      ;;
+    *)
+      echo "unknown argument $1"
+      exit 1
+      ;;
+  esac
+  shift
+done 
+
+echo "Turning [$POWER] services for project [$PROJECT] tagged [$TAG_ENV]"
+
```

Comment:
There are services that either have complete data loss (e.g. Kafka) or run the risk of partial data loss (e.g. Redis) when being powered off. Is a confirmation dialog that acknowledges this risk important here?

Reply:
Oh right! Why not. Let's go for a confirmation dialog then!